docs(observability): correct TraceQL syntax in query reference and troubleshooting guide (#3274)
giantswarm/docs: src/content/overview/observability/data-management/data-exploration/troubleshoot-traces-with-traceql/index.md

1. **Find error traces** for the affected service:

   ```traceql
   {span:status = error && resource.service.name = "payment-service"}
   ```

> 2. **Narrow down by time** to the window where the problem occurred. TraceQL has no time function; use Grafana's time range picker (top right of Explore) to bound the same query to the incident window.

Reviewer: ```suggestion
2. **Narrow down by time** to the window where the problem occurred with Grafana's time range picker.
```
